### ISSUE TITLE
bug: scroll assist activates in playground iframes

### ISSUE THREAD

**Issue by chrissmejia** [ionic-team/ionic-docs#3416]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

v7.x

### Current Behavior

Content is pushed up on input focus on emulated chrome device

UPDATE: It happens on Android too, the extra space is also present when the Android native keyboard is pushed up

### Expected Behavior

Don't move

### Steps to Reproduce

If you go to the oficial documentation and turn on the device toolbar and focus an input, the content is pushed up

Without focus:
<img width="442" alt="Screenshot 2024-01-13 at 11 58 52" src="https://github.com/ionic-team/ionic-framework/assets/3825991/94cd843f-d7df-4d13-9289-f67beedeac96">

With focus:
<img width="465" alt="Screenshot 2024-01-13 at 11 59 01" src="https://github.com/ionic-team/ionic-framework/assets/3825991/dcdf38cf-fcb1-40b8-ab3e-25b1241bbe67">

### Code Reproduction URL

https://ionicframework.com/docs/api/input

### Ionic Info

Same as in the docs, I'm personally having the same issue on "@ionic/vue": "^7.6.3",

### Additional Information

I'm migrating from 6.x to 7.x, 6.x was fine.

**Comment by chrissmejia** [comment]
> Thanks! I am going to transfer this to the docs repo. We have a scroll assist utility that scrolls content to prevent inputs from being hidden by the keyboard. However, we should probably disable this in the iframe demos on the docs.

Thanks a lot for your support, can you please tell me how to disable that for my app? Because it's happening since we migrated from 6.x to 7.x

**Comment by liamdebeasi** [comment]
You can set `scrollAssist: false` in your IonicConfig. However, I don't recommend doing that because then text inputs may be hidden by the keyboard.

**Comment by chrissmejia** [comment]
> You can set `scrollAssist: false` in your IonicConfig. However, I don't recommend doing that because then text inputs may be hidden by the keyboard.

Thanks a lot for the feedback, I understand but I still have a bug in the system. if I remove 

```
<style lang="scss">
    ion-content {
       --keyboard-offset: 0px !important;
    }
</style>
```
I get the UI jumping with empty space even in Android that it's a terrible experience, same than in the docs, something changed between 6.x and 7.x in this sense? because it was working well, I need guidance to fix the issue.

**Comment by chrissmejia** [comment]
Sure, thank you so much for the context, my goal with your help is to debug it and fix it, so I'm here for anything you require.

https://github.com/ionic-team/ionic-docs/assets/3825991/922b51ed-916d-4301-ad49-c04980f45016

In real life I want the complete view moving up if require to show the keyboard no just padding in the bottom, is that posible?

**Comment by liamdebeasi** [comment]
> In real life I want the complete view moving up if require to show the keyboard no just padding in the bottom, is that posible?

Yes, but that would be something the developer is responsible for. You'd want to listen for the keyboard open/close events and transform the container as shown in this demo: https://ionic.io/blog/keyboard-improvements-for-ionic-apps

**Comment by vochitan90** [comment]
Any update?
